Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -5414,10 +5499,8 @@ def __init__(self, outputs_prefix=None, outputs_key_field=None, outputs=None, in
         self.indicators_timeline = indicators_timeline
         self.ignore_auto_extract = ignore_auto_extract
         self.mark_as_note = mark_as_note
-        self.polling_command = polling_command
-        self.polling_args = polling_args
-        self.polling_timeout = polling_timeout
-        self.polling_next_run = polling_next_run
+        self.scheduled_command = scheduled_command
+        self.disable_readable_output = disable_readable_output
```

Comment:
Actually, in these cases, it does return an output, but since it won't contain a true update, I can also just remove the output and deal with it as you suggested 👍